build-tests/heft-webpack5-everything-test/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "heft-webpack5-everything-test",
+  "description": "Building this project tests every task and config file for Heft when targeting the web browser runtime using Webpack 5",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "heft test --clean",
+    "start": "heft start"
+  },
+  "devDependencies": {
+    "@rushstack/eslint-config": "workspace:*",
+    "@rushstack/heft": "workspace:*",
+    "@rushstack/heft-webpack5-plugin": "workspace:*",
+    "@types/heft-jest": "1.0.1",
+    "@types/webpack-env": "1.13.0",
+    "eslint": "~7.12.1",
+    "file-loader": "~6.0.0",
+    "tslint": "~5.20.1",
+    "tslint-microsoft-contrib": "~6.2.0",
+    "typescript": "~3.9.7",
+    "webpack": "~4.44.2"
```

Comment:
does this version of webpack not have an effect on that test? like the plugin loads it's own version and ignores this? and if so would this dep be needed IRL?

Reply:
Oops. I'll fix that.